python/src/aac/plugins/first_party/lsp_server/providers/semantic_tokens_provider.py

```diff
@@ -37,6 +37,7 @@ def _is_struct(token: Token, language_context: LanguageContext) -> bool:
 
 def _is_string(token: Token, language_context: LanguageContext) -> bool:
     """Return whether the provided token represents a string value."""
+    _ = language_context  # Keeping the argument to adhere to the interface
```

Comment:
Can you help me understand why we're doing this? Is it to get rid of a warning that `language_context` is unused, or something? If so, I think it would be better to rename `language_context` to `_` in the parameter list as that specifically indicates we aren't using the value in the function and it achieves the same thing we're doing with this line.

Reply:
I didn't know you could do that in Python. Neat!